hasher-matcher-actioner/Dockerfile

```diff
@@ -5,9 +5,7 @@ RUN yum install -y gcc gcc-c++
 
 # Install dependencies as a separate step so layers are reuseable.
 COPY setup.py ./
-RUN python setup.py egg_info \
-    && pip install -r hmalib.egg-info/requires.txt \
-    && rm -r hmalib.egg-info setup.py
+RUN pip install -e .
```

Comment:
@schatten before you ask. I checked this was just as fast for hmalib change updates and still in its own layer. The old command broke because  `hmalib.egg-info/requires.txt` actually have more context from `setup.py` then a `requirements.txt`  (https://setuptools.readthedocs.io/en/latest/deprecated/python_eggs.html#requires-txt)